src/services/utilities.ts

```diff
@@ -1840,6 +1840,10 @@ namespace ts {
         return program.getSourceFiles().some(s => !s.isDeclarationFile && !program.isSourceFileFromExternalLibrary(s) && !!s.externalModuleIndicator);
     }
     export function compilerOptionsIndicateEs6Modules(compilerOptions: CompilerOptions): boolean {
+        // Prevent ES6 module indication for non-module files
+        if(compilerOptions.module === ModuleKind.None){
+            return false;
+        }

```

Comment:
1. spacing
2. `--module none` does _not_ mean files are “non-modules.” It honestly [doesn’t mean anything at all](https://github.com/microsoft/TypeScript/issues/48790).

```suggestion
        if (compilerOptions.module === ModuleKind.None) {
            return false;
        }
```

Reply:
> 1. spacing
> 
>     2. `--module none` does _not_ mean files are “non-modules.” It honestly [doesn’t mean anything at all](https://github.com/microsoft/TypeScript/issues/48790).

On it

Reply:
> 1. spacing
> 
>     2. `--module none` does _not_ mean files are “non-modules.” It honestly [doesn’t mean anything at all](https://github.com/microsoft/TypeScript/issues/48790).

Do you have any suggestions or heads up of where i can the file module configurations from instead of compilerOptions?

Reply:
removed the comment , you can check now @andrewbranch :)